epitome/dataset.py

```diff
@@ -21,16 +21,18 @@
 from sklearn.metrics import jaccard_score
 
 # local imports
-from epitome import *
 from .constants import Dataset
 from .functions import download_and_unzip
 from .viz import plot_assay_heatmap
 
 ################### File accession constants #######################
-S3_DATA_PATH = 'https://epitome-data.s3-us-west-1.amazonaws.com/hg19.zip'
+S3_DATA_PATH = 'https://epitome-data.s3-us-west-1.amazonaws.com'
 
-# os env that should be set by user to explicitly set the data path
-EPITOME_DATA_PATH_ENV="EPITOME_DATA_PATH"
+# List of available assembiles in S3_DATA_PATH
+EPITOME_GENOME_ASSEMBLIES = ['hg19', 'hg38', 'test']
+DEFAULT_EPITOME_ASSEMBLY = "hg19"
+EPITOME_USER_PATH = os.path.join(os.path.expanduser('~'), '.epitome')
+DEFAULT_EPITOME_DATA_PATH = os.path.join(EPITOME_USER_PATH, 'data')
```

Comment:
can you comment each of these variables?

Reply:
what do you mean by comment the variables?